### DISCUSSION TITLE
[Feature Request] Download from Subsonic

### DISCUSSION THREAD

**Discussion by cproensa** [strawberrymusicplayer/strawberry#1271] [Ideas]
I have all the music in a external subsonic server, and many times, when listening to songs in Strawberry i would like to download the song to my local. For example, some files that i want to add to a (playlist) folder to later copy to a usb to listen in the car, etc.

With the local collection, there is an option "show in file browser", which allows you to quickly find and pick that specific file. But with subsonic collection, there is not any similar option, afaict.

It would be nice to have an option, eg in contextual menu, for "download file".
Subsonic API has a "download" method.

Additional thougths about?
- how to name a downloaded file, based in artist, title, etc...
- A download action allowed for a selection of multiple songs?

**Comment by kpmgeek** [comment]
This would be great, especially if there was an option to convert while downloading to a lossy format.

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
